conanfile.py

```diff
@@ -0,0 +1,30 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
```

Comment:
In general, in all conan files, let's keep exactly one blank line between sections, and one single space between multiple arguments separated by commas (,)

Reply:
handled